packages/medusa-react/src/helpers/index.ts

```diff
@@ -126,7 +126,7 @@ export const formatAmount = ({
 }
 
 // we should probably add a more extensive list
-const noDivisionCurrencies = ["krw", "jpy", "vnd"]
+const noDivisionCurrencies = ["pkr","krw", "jpy", "vnd"]
```

Comment:
Shouldn't this list be the exactly the same as the one in `packages/medusa-core-utils/src/zero-decimal-currencies.ts`?

Reply:
yeah, it needs to be.

Reply:
Do you think it's worth doing the change on this PR (importing and reusing `noDivisionCurrencies`) or it's out of scope? cc @srindom